source/mainnet/net/snippets/delegation-faq.rst

```diff
@@ -0,0 +1,98 @@
+
+The second cap is for security reasons as it prevents a party with a small stake from controlling too large a part of the system using leverage.
+
+**What happens when the maximum stake cap is reached for my baker?**
+These two caps are soft caps in the sense that a pool can have more stake than allowed, but only the allowed amounts are taken into account in the consensus algorithm and when distributing rewards. A pool violating the amount limits will gradually receive less rewards per CCD in the pool as the size of the pool continues to increase.
+
+**How much less/more can I expect to earn when opening my pool to delegators?**
+The amount you earn depends on how much stake your pool is baking. The more delegators, the more stake you have and the greater your chances of being selected to bake a block are. 10% of the additional rewards the delegators’ stake in your pool attracts will go to the pool owner. As you are able to have 2x your own stake delegated to your pool, you are effectively able to increase your rewards by 20% if your pool is full.
+
+Therefore, attracting delegators and managing your baker responsibly will maximize rewards for all. In :ref:`Baker management<baker-pool>` you'll find information about how to be a responsible baker.
+
+**Is there a cool-down period for bakers?**
+Yes, the cool-down period for bakers is three weeks when reducing stake or stopping baking.
+
+**Can I change my restaking preference if I am in a cool-down period?**
+Yes, you can. Note that anything that is restaked during the cool-down period will be unstaked after the stake reduction takes effect.
+
+**How much does it cost to begin baking?**
+There is no specific fee to begin baking, however, you do need to stake a minimum of 14,000 CCD, and you need to have enough CCD left in your disposable balance to cover transaction fees to generate your baker keys (link to definition?).
+
+
+For delegators
+--------------
+
+**Can I delegate to more than one baker at a time?**
+Yes. You can delegate to multiple bakers but an account can only have one delegation. To delegate to multiple bakers, you need multiple accounts.
+
+**Can I move from one baker pool to another without having to wait for the next pay day?**
+Changes to the pools take effect every 24 hours. So increasing the stake, moving the stake between pools or between passive delegation and a baking pool all take effect at the :ref:`pay day<glossary-pay-day>`.
+
+**How much does it cost to begin delegating?**
+There is no minimum amount required to delegate stake to a pool or enable passive delegation. You just have to have enough CCD left in your disposable balance to cover transaction fees.
+
+**How much can I expect to earn as a delegator?**
+This depends on the baker pool you select and how reliable this baker is. It also depends on how much stake the baker has. It is important to use `CCDScan <https://ccdscan.io>`_ to research bakers before delegating stake.
```

Comment:
Is it obvious enough that peoples delegation amount plays a role, or should that be mentioned?